neurons/validators/src/core/validator.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, debug_miner=None):
         self.should_exit = False
         self.is_running = False
         self.last_job_run_blocks = 0
+        self.set_weights_success = True
```

Comment:
@LegendWorrier it should be False for the initial data, but  I don't understand why we need this...